sos/report/plugins/openstack_heat.py

```diff
@@ -52,7 +52,24 @@ def setup(self):
                                     "intended to connect to the OpenStack "
                                     "environment.")
             else:
-                self.add_cmd_output("openstack stack list")
+                self.add_cmd_output("openstack stack list --all-projects")
+                self.add_cmd_output("openstack stack list --all-projects "
+                                    "--nested")
+
+                cmd = "openstack stack list -f value --all-projects"
+                res = self.exec_cmd(cmd)
+                if res['status'] == 0:
+                    heat_stacks = res['output']
+                    for stack in heat_stacks.splitlines():
+                        stack = stack.split()[0]
+                        cmd = f"openstack stack show {stack}"
+                        self.add_cmd_output(
+                            cmd,
+                            suggest_filename=f"stack-{stack}.log")
+                        cmd = f"openstack stack resource list {stack} -n 10"
+                        self.add_cmd_output(
+                            cmd,
+                            suggest_filename=f"stack-resources-{stack}.log")
```

Comment:
Ditto. The command naming seems clear and human friendly enough, not sure why we're casting these as log files instead of the command output.